MongoDB/include/Poco/MongoDB/MessageHeader.h

```diff
@@ -38,14 +38,24 @@ class MongoDB_API MessageHeader
 
 	enum OpCode
 	{
+#if false
```

Comment:
I guess this #if should be removed. 
Generally speaking, I wonder how much warnings this whole PR generates by default?

Reply:
Issue to resolve MongoDB deprecate functionality is #4603. That is a quite big task on its own.

The PR does not generate more warnings as code from devel, probably much less.